src/vehicle_use.cpp

```diff
@@ -1966,6 +1967,12 @@ void vehicle::use_bike_rack( int part )
         map &here = get_map();
         here.invalidate_map_cache( here.get_abs_sub().z );
         here.rebuild_vehicle_level_caches();
+
+        //ensures that the turn timer used as id is moved along after each successfull bike rack interaction
+        const activity_id ACT_WAIT( "ACT_WAIT" );
+        activity_id actType = ACT_WAIT;
+        player_activity new_act( actType, 100 * ( to_turns<int>( 5_minutes ) ), 0 );
+        get_player_character().assign_activity( new_act, false );
```

Comment:
You should add a specific vehicle racking activity, so it doesn't say `stop waiting` if you try to interrupt it.
And this should probably trigger at the end of that activity.

Reply:
I don't think that would work properly.